tox.ini

```diff
@@ -10,11 +10,14 @@ envdir =
     py37: {toxworkdir}/3.7
     py38: {toxworkdir}/3.8
     py39: {toxworkdir}/3.9
+    py310: {toxworkdir}/3.10
```

Comment:
Early forays into 3.10. This is actually excluded from the envlist, so just for experimental testing purposes.